src/Native/Runtime/gcheaputilities.h

```diff
@@ -20,9 +20,26 @@ GPTR_DECL(uint32_t,g_card_table);
 }
 #endif // !DACCESS_COMPILE
 
+extern "C" uint32_t* g_card_bundle_table;
```

Comment:
It would be nice to put this under `#ifdef FEATURE_MANUALLY_MANAGED_CARD_BUNDLES` as well.